starter_code/app.js

```diff
@@ -15,14 +15,32 @@ app.use(express.static(__dirname + '/public'));
 
 
 // setting the spotify-api goes here:
+const spotifyApi = new SpotifyWebApi({
+  clientId: process.env.CLIENT_ID,
+  clientSecret: process.env.CLIENT_SECRET
+});
 
+// Retrieve an access token
+spotifyApi
+  .clientCredentialsGrant()
+  .then(data => {
+    spotifyApi.setAccessToken(data.body["access_token"]);
+  })
+  .catch(error => {
+    console.log("Something went wrong when retrieving an access token", error);
+  });
 
 
 
 
 
 // the routes go here:
+app.get('/', (req, res, next) => {
+  res.render('index');
+})
 
-
+app.get('/search', (req,res,next)=>{
```

Comment:
Fanny, je te donne la première route pour qu'elle te serve d'exemple pour les autres itérations : 
app.get('/artists', function (req, res, next) {
  spotifyApi
  .searchArtists(`${req.query.artist}`)
  .then(data => {
    console.log("The received data from the API: ", data.body);
    // ----> 'HERE WHAT WE WANT TO DO AFTER RECEIVING THE DATA FROM THE API'
    res.render('artists', {
      artists: data.body.artists.items
    });
  })
  .catch(err => {
    console.log("The error while searching artists occurred: ", err);
  });
});